rules/aws_cloudtrail_rules/aws_iam_group_read_only_events.py

```diff
@@ -0,0 +1,42 @@
+from panther_base_helpers import aws_rule_context
+
+
+def rule(event):
+    arn_whitelist = []
```

Comment:
I would prefer for this word to be spelled `arn_allowlist` , and for it to be established as a global in the detection ala [aws_security_configuration_change.py](https://github.com/panther-labs/panther-analysis/blob/master/rules/aws_cloudtrail_rules/aws_security_configuration_change.py#L19-L22)